components/dialog/metadata/mods.md

```diff
@@ -0,0 +1,27 @@
+| Modifiable Custom Properties                   |
+| ---------------------------------------------- |
+| `--mod-dialog-confirm-border-radius`           |
+| `--mod-dialog-confirm-buttongroup-padding-top` |
+| `--mod-dialog-confirm-close-button-padding`    |
+| `--mod-dialog-confirm-close-button-size`       |
+| `--mod-dialog-confirm-description-margin`      |
+| `--mod-dialog-confirm-description-padding`     |
+| `--mod-dialog-confirm-description-text-size`   |
+| `--mod-dialog-confirm-divider-height`          |
+| `--mod-dialog-confirm-footer-padding-top`      |
+| `--mod-dialog-confirm-gap-size`                |
+| `--mod-dialog-confirm-hero-height`             |
+| `--mod-dialog-confirm-large-width`             |
+| `--mod-dialog-confirm-medium-width`            |
+| `--mod-dialog-confirm-padding-200`             |
+| `--mod-dialog-confirm-padding-300`             |
+| `--mod-dialog-confirm-padding-grid`             |
+| `--mod-dialog-confirm-small-width`             |
+| `--mod-dialog-confirm-title-text-font-weight`  |
+| `--mod-dialog-confirm-title-text-size`         |
+| `--mod-dialog-fullscreen-header-text-size`     |
+| `--mod-dialog-min-inline-size`                 |
+| `--mod-gray-800`                               |
+| `--mod-gray-900`                               |
+| `--mod-line-height-100`                        |
```

Comment:
We shouldn't have mod names that are the same as the global token names. It looks like we lost two color variable names from the original skin.css. Can we redefine these at the top and use them in the CSS?
`--spectrum-dialog-confirm-title-text-color`
`--spectrum-dialog-confirm-description-text-color`

For line-height, it looks like there also was
`--spectrum-dialog-confirm-title-text-line-height`
`--spectrum-dialog-confirm-description-text-line-height`
Can we define those as well? They should both have their own mods.

Reply:
done!